packages/unity-bootstrap-theme/stories/docs/GetStarted/get-started.stories.mdx

```diff
@@ -0,0 +1,91 @@
+5. Test installing packages using yarn or npm inside of another NPM project:
+```yarn add @asu/design-tokens```
+
+### TROUBLESHOOTING INSTALLATION ERRORS
+
+If you get errors having to do with yarn not being able to find a package on the registry, try running ```yarn config list``` at the project root and look for the ```registry:``` key under yarn config. If it is not set to ```https://registry.yarnpkg.com``` then run ```yarn config delete registry``` and recheck config.
+
+
+## Advanced Details for Unity Developers and Contributors
+
+See [this section](https://github.com/ASU/asu-unity-stack#advanced-details-for-unity-developers-and-contributors) of our README for more information about the Unity monorepo and how to contribute to the project.
+
+## > Google Analytics integration
+
+Google Analytics integration for Unity is handled via Google Tag Manager. During user interaction with Unity components, usage data is captured by Google Tag Manager and delivered to Google Analytics.
+
+This is enabled using the window `dataLayer` object. For components in each package of this repository we take the `dataLayer` object, with GTM and the dataLayer having already been initialized (by the host application or CMS - see [https://unity.web.asu.edu](https://unity.web.asu.edu) for details on GTM setup), and push events using the `push()` method of the `dataLayer`. For each object included in the array, Google Tag manager catches that event.
+
+Depending on the package, this integration, of dispatching events, is handled in different ways. For `component-header`, `components-library` header and `component-footer`, we use a service that pushes each event if the `dataLayer` object exists and that service method is called on each jsx element event handler. For example:
+
+#### **`src/component.js`**
+```JS
+<a href="#" onFocus={() => trackGAEvent(customEvent)}>Anchor Text</a>
+```
+#### **`services/googleAnalytics.js`**
+```JS
+const trackGAEvent = (event) => {
+  const { dataLayer } = window;
+  if (dataLayer) dataLayer.push(event);
+}
+```
+
+For the `unity-bootstrap-theme` package the events are dispatched by an `eventListener`, for the `focus`, `click` or `change` event handler, for each html element that needs to be included. For example:
+
+#### **`src/component.html`**
+```JS
+<a href="#" data-ga="">Anchor Text</a>
+```
+#### **`src/component.js`**
+```JS
+const pushGAEvent = (event) => {
+  const { dataLayer } = window;
+  if (dataLayer) dataLayer.push(event);
+};
+// eventListener
+const elements = document.querySelectorAll('[data-ga]');
+elements.forEach((element) =>
+  element.addEventListener('focus', () => {
+    pushGAEvent(event);
+  })
+);
+```
+Check [https://unity.web.asu.edu/](https://unity.web.asu.edu/) as well as the README.md's in individual Unity packages for more Google Analytics, Google Tag Manager and data layer implementation details.
```

Comment:
In this PR, the `unity-bootstrap-theme` package's [README.txt](https://github.com/ASU/asu-unity-stack/blob/UDS-1421/packages/unity-bootstrap-theme/README.md) is still rough, and the get-started.stories.mdx content would be a good replacement for that... Is there any way we could use the same content for both - such as move this content into the README.md, and then for the Get Started documentation story, pull that content in? Just trying to push us a little harder to get two birds with one stone (and have docs that are easier to maintain).

Reply:
I am not sure if this addon will solve the issue, but it looks promising, that way the MDX file is still the only file https://github.com/storybookjs/storybook/tree/master/addons/docs#mdx
(If I understood the purpose of this addon correctly)